src/qt/transactionview.cpp

```diff
@@ -446,6 +450,21 @@ void TransactionView::abandonTx()
     model->getTransactionTableModel()->updateTransaction(hashQStr, CT_UPDATED, false);
 }
 
+void TransactionView::resendTx()
+{
+    if(!transactionView || !transactionView->selectionModel())
+        return;
+    QModelIndexList selection = transactionView->selectionModel()->selectedRows(0);
```

Comment:
should we handle if multiple txes if they are selected? I think we probably should. Right now if you select two txes where the first is not resend-able but the second one isn't it won't let you resend, but if you select two resend-able txes and click resend it'll only do one, or anytime someone selects two we don't let you resend? not sure

Reply:
Good point! 👍  The idea was to only let resend them one by one and not in batches so I think it makes sense to disable the menu item when 2 or more rows are selected. db9f0d6e